epoxy-processor/src/main/java/com/airbnb/epoxy/AttributeInfo.java

```diff
@@ -1,251 +1,51 @@
 package com.airbnb.epoxy;
 
-import com.airbnb.epoxy.EpoxyAttribute.Option;
 import com.squareup.javapoet.AnnotationSpec;
-import com.squareup.javapoet.ClassName;
 import com.squareup.javapoet.TypeName;
 
-import java.lang.annotation.ElementType;
-import java.lang.annotation.Target;
 import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.HashSet;
 import java.util.List;
-import java.util.Set;
 
-import javax.lang.model.element.AnnotationMirror;
-import javax.lang.model.element.Element;
-import javax.lang.model.element.ElementKind;
-import javax.lang.model.element.ExecutableElement;
-import javax.lang.model.element.Modifier;
-import javax.lang.model.element.TypeElement;
-import javax.lang.model.type.DeclaredType;
-import javax.lang.model.util.Types;
+import javax.lang.model.type.TypeMirror;
 
-import static com.airbnb.epoxy.ProcessorUtils.capitalizeFirstLetter;
 import static com.airbnb.epoxy.ProcessorUtils.isViewClickListenerType;
-import static com.airbnb.epoxy.ProcessorUtils.startsWithIs;
-import static javax.lang.model.element.Modifier.FINAL;
-import static javax.lang.model.element.Modifier.PRIVATE;
-import static javax.lang.model.element.Modifier.PROTECTED;
-import static javax.lang.model.element.Modifier.PUBLIC;
-import static javax.lang.model.element.Modifier.STATIC;
 
 class AttributeInfo {
```

Comment:
I'm indifferent about this as it is `package private`, and maybe @elihart some sort of potential use case for not making it `abstract`. Also, it seems a bit odd to me to make a simple bean/dto abstract, but then again I do see where you are coming from geralt 😄 .